packages/fxa-auth-server/lib/metrics/glean/index.ts

```diff
@@ -155,7 +155,8 @@ export function gleanMetrics(config: ConfigType) {
     },
 
     oauth: {
-      tokenCreated: createEventFn('oauth_token_created'),
+      tokenCreated: createEventFn('access_token_created'),
```

Comment:
I updated the Glean event name for consistency.  The event hasn't been deployed anywhere yet.